packages/fork-choice/src/protoArray/protoArray.ts

```diff
@@ -1,12 +1,16 @@
 import {Epoch, RootHex, Slot} from "@lodestar/types";
 import {computeEpochAtSlot} from "@lodestar/state-transition";
+import {toHexString} from "@chainsafe/ssz";
 
-import {ProtoBlock, ProtoNode, HEX_ZERO_HASH} from "./interface.js";
-import {ProtoArrayError, ProtoArrayErrorCode} from "./errors.js";
+import {ProtoBlock, ProtoNode, HEX_ZERO_HASH, ExecutionStatus, LVHExecResponse} from "./interface.js";
+import {ProtoArrayError, ProtoArrayErrorCode, LVHExecError, LVHExecErrorCode} from "./errors.js";
 
 export const DEFAULT_PRUNE_THRESHOLD = 0;
 type ProposerBoost = {root: RootHex; score: number};
 
+const ZERO_HASH = Buffer.alloc(32, 0);
+const ZERO_HASH_HEX = toHexString(ZERO_HASH);
```

Comment:
Just merge this two together
```suggestion
const ZERO_HASH_HEX = toHexString(Buffer.alloc(32, 0));
```